src/db/client.js

```diff
@@ -119,6 +121,16 @@ async function listRoutines(server, database) {
   return database.connection.listRoutines();
 }
 
+async function listTableColumns(server, database, table) {
```

Comment:
I guess need the async because of the https://github.com/BornaP/sqlectron-core/blob/13-fetch-table-columns/src/db/client.js#L172.
That throws an exception. I guess without the async it will not reject the promise properly.
Since that is the current pattern used for all functions in that file. I will accept without applying your suggestions. We can try out without async in another commit.